### PR TITLE
Integration of 0.14 submodule into ka-lite-central

### PR DESCRIPTION
**Pull request by jamalex**
Fixes to central server to integrate the 0.14.x code into ka-lite-submodule.

### CODE REVIEW

**Comment by jamalex** [comment]
@aronasorman I assigned this to you for now because @benjaoming isn't on this repo team yet, but he may be the better one to look at it (though I don't think he'll actually have time, I guess).

**Comment by jamalex** [comment]
This is now ready enough to start reviewing/testing. Some tests are failing, but the overall test framework and server should now be not totally broken.

**Comment by jamalex** [comment]
Travis file updated. Hopefully that helps get the tests up and running (though we do expect some will fail).

File: centralserver/contact/tests/unicode_tests.py [hunk 1]
```diff
@@ -3,19 +3,21 @@
...
+    # TODO(jamalex): this is broken because test_unicode_class_coverage no longer exists anywhere
+    # @unittest.skipIf(sys.version_info < (2,7), "Test requires python version >= 2.7")
+    # def test_unicode_class_coverage(self):
+    #     # Make sure we're testing all classes
+    #     self.check_unicode_class_coverage(
+    #         models_module="contact.models",
+    #         known_classes = [Contact, Contribute, Deployment, Info, Support],
+    #     )
 
+    korean_string = unichr(54392)
```
Comment (aronasorman) [inline centralserver/contact/tests/unicode_tests.py]: What's this line for?

Comment (aronasorman) [inline centralserver/contact/tests/unicode_tests.py]: Oh, it was defined in `UnicodeModelsTests` beforehand?

**Comment by aronasorman** [comment]
Test errors are mostly timeouts, however there's this one that caught my eye:

```
======================================================================
ERROR: test_loaddata (centralserver.central.tests.fixture_tests.FixtureTestCases)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/fle-internal/ka-lite-central/centralserver/central/tests/fixture_tests.py", line 19, in test_loaddata
    out = call_command("loaddata", fixture_file)
  File "/home/travis/build/fle-internal/ka-lite-central/ka-lite-submodule/python-packages/django/core/management/__init__.py", line 161, in call_command
    return klass.execute(*args, **defaults)
  File "/home/travis/build/fle-internal/ka-lite-central/ka-lite-submodule/python-packages/django/core/management/base.py", line 263, in execute
    output = self.handle(*args, **options)
  File "/home/travis/build/fle-internal/ka-lite-central/ka-lite-submodule/python-packages/django/core/management/commands/loaddata.py", line 224, in handle
    connection.check_constraints(table_names=table_names)
  File "/home/travis/build/fle-internal/ka-lite-central/ka-lite-submodule/python-packages/django/db/backends/sqlite3/base.py", line 341, in check_constraints
    referenced_table_name, referenced_column_name))
IntegrityError: Problem installing fixtures: The row in table 'central_organization' with primary key '1' has an invalid foreign key: central_organization.owner_id contains a value '1' that does not have a corresponding value in auth_user.id.
```

File: centralserver/contact/tests/unicode_tests.py (hunk 1 above)
Comment (jamalex) [inline centralserver/contact/tests/unicode_tests.py]: Yeah, this was the one thing needed to get it working without that inheritance.

**Comment by jamalex** [comment]
Got it down to just the timeout errors and a weird i18n encoding error. Merging.

**Pull request merged**